cli/glnexus_cli.cc

```diff
@@ -571,6 +572,93 @@ int main_genotype(int argc, char *argv[]) {
     return 0;
 }
 
+int main_iter_compare(int argc, char *argv[]) {
+    if (argc == 2) {
```

Comment:
It seems like there aren't any arguments being used, so the argument handling code here (L576-605) could just be deleted? Another issue is the invocations of help_init() will show the wrong help message.